recipe/meta.yaml

```diff
@@ -38,16 +51,19 @@ requirements:
 test:
   commands:
     - test -f $PREFIX/include/lemon/glpk.h                 # [unix]
-    - test -f $PREFIX/lib/libemon.a                        # [unix]
+    - test -f $PREFIX/lib/libemon.so                       # [linux]
+    - test -f $PREFIX/lib/libemon.dylib                    # [osx]
     - if not exist %LIBRARY_INC%\\lemon\\glpk.h exit 1     # [win]
     - if not exist %LIBRARY_LIB%\\lemon.lib exit 1         # [win]
+    - if not exist %LIBRARY_LIB%\\lemon.dll exit 1         # [win]
 
 about:
   home: https://lemon.cs.elte.hu/trac/lemon
-  license: Boost-1.0
+  license: BSL-1.0
```

Comment:
@k-dominik, this suffices to address gh-12 since the `license_file: LICENSE` below reference the `LICENSE` from the source tarball.
(The license file outside the `recipe` folder just states the license for the recipe itself but not the package.)

Reply:
thank you very much for clearing that up!